actions/images

```diff
@@ -0,0 +1,38 @@
+#!/usr/local/sbin/charm-env python3
+
+import sys
+from subprocess import check_output, CalledProcessError, STDOUT
+
+from charmhelpers.core import hookenv
+from charms import layer
+from charms.reactive import is_flag_set
+
+layer.import_layer_libs()
+
+# start reactive
+hookenv._run_atstart()
+
+
+def fail(msg):
+    hookenv.action_fail(msg)
+    sys.exit()
+
+
+if not is_flag_set('charm.docker-registry.configured'):
+    fail('Docker Registry charm is not yet ready.')
+
+cmd = ['docker', 'images']
+options = hookenv.action_get('options')
+if options:
+    cmd.extend(options.split())
```

Comment:
I always use `shlex.split()` for command-line strings.

Reply:
Yikes.  I even touted the `options` param as 'gimme anything you'd give to the shell'!  Nice catch.